### DISCUSSION TITLE
Does MMOCRInferencer work for SVTR-base?

### DISCUSSION THREAD

**Discussion by zwg210** [open-mmlab/mmocr#1959] [Q&A]
The 'model' dictionary does not fit the requirements as it lacks 'type', tried working around but did not succeed. What are the models the inferencer is compatible with?

**Comment by gaotongxiao** [comment]
I didn't get your question. Here is the list of supported models in inferencer:

https://mmocr.readthedocs.io/en/dev-1.x/modelzoo.html#overview

where `svtr-base` is included. If you have encountered any error, you may raise an issue and provide sufficient context about it.
